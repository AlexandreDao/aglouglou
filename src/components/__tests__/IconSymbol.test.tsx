import React from 'react'
import { render } from '@testing-library/react-native'
import { IconSymbol as IconSymbolAndroid } from '@/components/IconSymbol.android'
import { IconSymbol as IconSymbolIOS } from '@/components/IconSymbol.ios'

describe('<IconSymbol />', () => {
  test('IconSymbol snapshot (Android)', () => {
    const tree = render(<IconSymbolAndroid size={28} name="star.fill" color="white" />).toJSON()

    expect(tree).toMatchSnapshot()
  })

  test('IconSymbol snapshot (iOS)', () => {
    const tree = render(<IconSymbolIOS size={28} name="star.fill" color="white" />).toJSON()

    expect(tree).toMatchSnapshot()
  })
})
