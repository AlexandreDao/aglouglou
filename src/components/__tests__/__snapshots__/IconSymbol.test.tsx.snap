// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<IconSymbol /> IconSymbol snapshot (Android) 1`] = `
<Text
  allowFontScaling={false}
  selectable={false}
  style={
    [
      {
        "color": "white",
        "fontSize": 28,
      },
      undefined,
      {
        "fontFamily": undefined,
        "fontStyle": "normal",
        "fontWeight": "normal",
      },
      {},
    ]
  }
>
  î ¸
</Text>
`;

exports[`<IconSymbol /> IconSymbol snapshot (iOS) 1`] = `
<ViewManagerAdapter_SymbolModule
  animated={false}
  colors={[]}
  name="star.fill"
  resizeMode="scaleAspectFit"
  style={
    [
      {
        "height": 24,
        "width": 24,
      },
      [
        {
          "height": 28,
          "width": 28,
        },
        undefined,
      ],
    ]
  }
  tintColor="white"
  type="monochrome"
  weight="regular"
/>
`;
